config/backends.json

```diff
@@ -15,7 +18,10 @@
         "waterbutlerUrl": "https://staging-files.osf.io/",
         "helpUrl": "http://help.osf.io",
         "cookieLoginUrl": "https://staging-accounts.osf.io/login",
-        "oauthUrl": "https://staging-accounts.osf.io/oauth2/authorize"
+        "oauthUrl": "https://staging-accounts.osf.io/oauth2/authorize",
```

Comment:
Please also add corresponding property names to the `env` section (line 65) so that these new URLs can optionally be overridden as environment variables.